contracts/erc20-bridge-sampler/contracts/src/ERC20BridgeSampler.sol

```diff
@@ -807,4 +793,53 @@ contract ERC20BridgeSampler is
             );
         }
     }
+
+    function _sampleSellFromKyberNetwork(
+        address takerToken,
+        address makerToken,
+        uint256 takerTokenAmount
+    )
+        private
+        view
+        returns (uint256 makerTokenAmount)
+    {
+        address _takerToken = takerToken == _getWethAddress() ? KYBER_ETH_ADDRESS : takerToken;
+        address _makerToken = makerToken == _getWethAddress() ? KYBER_ETH_ADDRESS : makerToken;
+        uint256 takerTokenDecimals = _getTokenDecimals(takerToken);
+        uint256 makerTokenDecimals = _getTokenDecimals(makerToken);
+        (bool didSucceed, bytes memory resultData) = _getKyberNetworkProxyAddress().staticcall.gas(DEFAULT_CALL_GAS)(
+            abi.encodeWithSelector(
+                IKyberNetwork(0).kyberNetworkContract.selector
+            ));
+        if (!didSucceed) {
+            return makerTokenAmount;
+        }
```

Comment:
It's probably safe to just do a regular `address kyberNetworkContract = IKyberNetwork(_getKyberNetworkProxyAddress()).kyberNetworkContract();`. Kyber is on kovan, right?

Reply:
It's not in ganache